mentoring/programs/office-hours.md

```diff
@@ -0,0 +1,35 @@
+    * [Enhancements Subproject Meeting - Biweekly Tuesday at 08:30 PT (Pacific Time)](https://bit.ly/k8s-enhancements-agenda)
+    * [Production Readiness Office Hours - Biweekly Wednesday at 12:00 PT (Pacific Time)](https://docs.google.com/document/d/10QkXwiZfBL7wBKHHFslooXvGDKHA75jkkgiIXuKFryM/edit?usp=sharing)
+    * [Code organization Office Hours - Biweekly Thursday at 14:00 PT (Pacific Time)](https://docs.google.com/document/d/1HtTI0rJEGP_MSf6eO87aCmx_tzpovPAAg7U2Zxwm8FE/edit#)
+    * [Conformance office Hours - Biweekly Tuesday at 19:00 UTC](https://docs.google.com/document/d/1W31nXh9RYAb_VaYkwuPLd1hFxuRX3iU0DmaQ4lkCsX8/edit#)
+* Cluster Lifecycle - 
+    * [Cluster API (AWS implementation) office hours - Biweekly Monday at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1ojR0jooQaHEuz3gGW6DxcZlsws9ZX7jHPC8walm1G3A/edit)
+    * [Cluster API office hours - Biweekly Wednesday at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1fQNlqsDkvEggWFi51GVxOglL2P1Bvo2JhZlMhm2d-Co/edit)
+    * [Cluster Addons meeting - Biweekly Tuesday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/10_tl_SXcFGb-2109QpcFVrdrfnVEuQ05MBrXtasB0vk/edit)
+    * [Image Builder office hours - Biweekly Wednesday at 11:00 PT (Pacific Time)](https://docs.google.com/document/d/1YIOD0Nnid_0h6rKlDxcbfJaoIRNO6mQd9Or5vKRNxaU/edit)
+    * [Kubespray Office Hours - Biweekly Wednesday at 08:00 PT (Pacific Time)](https://docs.google.com/document/d/1oDI1rTwla393k6nEMkqz0RU9rUl3J1hov0kQfNcl-4o/edit)
+    * etcdadm Office Hours - Biweekly Monday at 09:00 PT (Pacific Time)
+    * [kops Office Hours - Biweekly Friday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/12QkyL0FkNbWPcLFxxRGSPt_tNPBHbmni3YLY-lHny7E/edit)
+    * [kubeadm Office Hours - Biweekly Wednesday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/130_kiXjG7graFNSnIAgtMS1G8zPDwpkshgfRYS0nggo/edit)
+    * [minikube Office Hours - Biweekly Monday at 11:00 PT (Pacific Time)](https://docs.google.com/document/d/1jhfmL1gsgN39uCEgz5pW9tnIotFgHhxq2yfMK3KYE4w/edit)
+* Docs - 
+    * [Korean Team Meeting - Biweekly Thursday at 13:00 UTC](https://docs.google.com/document/d/1h5sMhBpPB5unJmBAS7KzDiPs-_eFQOu5o4UyHwMtFCA/edit)
+* Multicluster - 
+    * [Cluster Federation (KubeFed) Working Group - Biweekly Wednesday at 07:30 PT (Pacific Time)](https://docs.google.com/document/d/1v-Kb1pUs3ww_x0MiKtgcyTXCAuZlbVlz4_A9wS3_HXY/edit)
+* Network - 
+    * [SIG Network Meeting - Biweekly Thursday at 14:00 PT (Pacific Time)](https://docs.google.com/document/d/1_w77-zG_Xj0zYvEMfQZTQ-wPP4kXkpGD8smVtW_qqWM/edit)
+    * [Service APIs Meeting (APAC Friendly Time) - Biweekly Thursday at 16:30 PT (Pacific Time)](https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit)
+    * [Service APIs Meeting (EMEA Friendly Time) - Biweekly Thursday at 10:30 PT (Pacific Time)](https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit)
```

Comment:
```suggestion
```
I think these are all regular meetings^^;;; but may need to confirm the service api meetings.